website/addons/github/routes.py

```diff
@@ -4,134 +4,118 @@
 
 from website.addons.github import views
 
-settings_routes = {
+
```

Comment:
@harrismendell, before you leave (maybe today?) please find time to sit down with @brianjgeiger and discuss your routes.